src/socket.cpp

```diff
@@ -181,19 +237,36 @@ void CSocket::SendPacket ( const CVector<uint8_t>& vecbySendBuf, const CHostAddr
         // send packet through network (we have to convert the constant unsigned
```

Comment:
`CVector<T>` subclasses `std::vector<T>`, so can this just use vecbySendBuf.c_data() to get a const pointer rather than copying the vector?

Reply:
Maybe, but I didn't change that part of the code.